app/src/main/java/org/fossasia/phimpme/share/SharingActivity.java

```diff
@@ -772,7 +772,9 @@ private void openCaptionDialogBox() {
         captionEditText.setHint(R.string.description_hint);
         captionEditText.setHintTextColor(ContextCompat.getColor(this,R.color.grey));
         captionEditText.setSelectAllOnFocus(true);
-        captionEditText.setHighlightColor(ContextCompat.getColor(getApplicationContext(), R.color.cardview_shadow_start_color));
+        if(getBaseTheme() == ThemeHelper.DARK_THEME || getBaseTheme() == ThemeHelper.AMOLED_THEME){
+            captionEditText.setHighlightColor(ContextCompat.getColor(getApplicationContext(), R.color.accent_grey));
```

Comment:
Set the captionEditText color also as black so that it is visible on highlighting.

Reply:
It is visible right now as well. I think we don't need to set it to black. Further when the user enters the caption for the first time, There is no highlight so black fonts are not visible on black background. Try running this code change once, i think it is working as expected

Reply:
ok then.